packages/gatsby-theme/src/blocks/BlogHeadBlock/BlogHeadBlock.stories.mdx

```diff
@@ -0,0 +1,89 @@
+import { Meta, Story, Props, Preview } from "@storybook/addon-docs/blocks"
+import BlogHeadBlock from "."
+
+export const data = {
+  title: `Gatsby's long road to incremental builds with Kyle Mathews`,
+  credits: "Via Unsplash",
+  author: {
+    name: "Marco José Sánchez",
+    link: "/",
+    ariaLabel: "Marco",
+  },
+  date: "Julio 2, 2020",
+  image: {
+    src:
+      "https://3czfu91fpa5s34atq735lonm-wpengine.netdna-ssl.com/wp-content/uploads/2018/12/types-of-infrastructure.jpeg",
+    alt: "Roads",
+  },
+}
+
+export const blogHeadArgTypes = {
+  backgroundColor: {
+    description: "Background overlay color",
+    control: {
+      type: "color",
+      presetColors: ["#001133CC"],
+    },
+  },
+}
+
+<Meta title='blocks/BlogHeadBlock' component={BlogHeadBlock} />
```

Comment:
We should pass the `argTypes` object directly into this Meta tag. This applies it to all the stories in this file.